src/Destructurama.ByIgnoring/Destructurama.ByIgnoring.csproj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netstandard1.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard1.1</TargetFrameworks>
```

Comment:
@nblumhardt I would drop netstandard1.1

Reply:
This seems reasonable to me. If we do we might want to increment a major version since it's a breaking change. Although the constructor change to `DestructureByIgnoringPolicy` might warrant a major version bump on its own (I think it's public, right?).

Reply:
Another option - wait for Serilog v3 release updating dependencies and bumping major version here with all breaking changes you want.

Reply:
I mean I'd prefer to not wait for Serilog v3 ;-)

So forget I said anything haha

Reply:
@chase-miller Hi. I was away from this repo and all OSS for a long time but now I have time to proceed here. Please pull from dev and resolve conflicts. Then I'm fine to review/merge.